External/FEXCore/Source/Interface/Core/Interpreter/InterpreterOps.cpp

```diff
@@ -1943,7 +1944,7 @@ void InterpreterOps::InterpretIR(FEXCore::Core::InternalThreadState *Thread, uin
                 uint32_t Src = *GetSrc<uint8_t*>(SSAData, Op->Header.Args[0]);
                 Src <<= 24;
```

Comment:
Since std::countl_zero supports 8-bit and 16 bit types. The shifts here can probably be removed?

Reply:
Good catch. Amended. Since these have defined behaviors on zero, we can also collapse the if statements too.